### ISSUE TITLE
[Feature Request] Heading 3 button

### ISSUE THREAD

**Issue by elevatorz89** [cryptee/web-client#35]
**Is your feature request related to a problem? Please describe.**
In the keyboard shortcuts, there is a little hint reffering to the fact there is a "third heading" in terms of markdown syntax, however it does not exist within the formatting bar. Literally all other formatting has clickable icons.

**Describe the solution you'd like**
Simply put, a H3 button in the formatting bar.

**Additional context**
N/A

**Comment by johnozbay** [comment]
Hi there! Hahah good catch! 

In short, when I first finished the editor, it had no markdown support.
Then when I've added markdown support, I ended up adding H3 for markdown spec-compatibility and cross-compatibility. – but at the time the editor [looked and functioned much much more differently](https://crypt.ee/imgs/old/docs-osx.jpg) (_yay nostalgia_), so at that time I didn't want to break things further. 

Added now, and will be out with the next release. 

<img width="312" alt="image" src="https://user-images.githubusercontent.com/611338/71556010-786bf700-2a3b-11ea-8e37-706764df000c.png">

![h3](https://user-images.githubusercontent.com/611338/71556019-95a0c580-2a3b-11ea-9d01-e1e9557cd73d.png)
